api/v2.0/swagger.yaml

```diff
@@ -9088,6 +9096,11 @@ definitions:
         description: Whether or not to skip update pull time for scanner
         x-omitempty: true
         x-isnullable: true
+      banner_message:
+        type: string
+        description: The banner message for the UI.It is the stringified result of the banner message object
```

Comment:
It's better to provide a sample value for it.

Reply:
Updated